src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -3386,9 +3386,13 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
           )
 
           let dto = p.Settings |> Server.deserialize<FSharpConfigRequest>
-          let c = config |> AVal.force
-          let c = c.AddDto dto.FSharp
-          updateConfig c
+
+          match dto.FSharp with
+          | Some fsharpConfig ->
+            let c = config |> AVal.force
+            let c = c.AddDto fsharpConfig
+            updateConfig c
+          | None -> ()
```

Comment:
NIT: Should we prefer `Option.iter` ?

Reply:
Updated w/ force-push -- not sure about formatting style for Option.iter though

Reply:
ok, fantomas suggested me a style no problem :)